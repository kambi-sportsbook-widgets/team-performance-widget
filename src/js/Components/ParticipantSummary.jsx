import React from 'react';

const ParticipantSummary = ({ children }) => {
   return (
      <div className='kw-participantsummary'>
         { children }
      </div>
   );
};

ParticipantSummary.propTypes = {
   /**
    * Inner components
    */
   children: React.PropTypes.node.isRequired,
};

export default ParticipantSummary;
