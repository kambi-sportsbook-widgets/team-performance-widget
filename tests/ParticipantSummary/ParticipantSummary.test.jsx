/* eslint-env jest */
import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
import ParticipantSummary from '../../src/js/Components/ParticipantSummary';

let renderer;

describe('ParticipantSummary DOM rendering', () => {
   beforeEach(() => {
      renderer = ReactTestUtils.createRenderer();
   });
   it('renders correctly', () => {
      expect(renderer.render(
         <ParticipantSummary>
            <span>content</span>
         </ParticipantSummary>
      )).toMatchSnapshot();
   });
});
