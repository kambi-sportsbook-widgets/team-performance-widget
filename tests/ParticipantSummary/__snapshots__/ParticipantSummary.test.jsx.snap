// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParticipantSummary DOM rendering renders correctly 1`] = `
<div
  className="kw-participantsummary"
>
  <span>
    content
  </span>
</div>
`;
